editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -228,7 +204,60 @@ function collectSelectableSiblings(
 
     addChildrenAndUnfurledFocusedComponents(allPaths)
   })
-  return siblings
+
+  return replaceNonSelectablePaths(siblingsChildrenAndAncestors, componentMetadata, lockedElements)
+}
+
+function replaceNonSelectablePaths(
+  selectablePaths: Array<ElementPath>,
+  componentMetadata: ElementInstanceMetadataMap,
+  lockedElements: LockedElements,
+): Array<ElementPath> {
+  let updatedSelectablePaths: Array<ElementPath> = []
+  Utils.fastForEach(selectablePaths, (selectablePath) => {
+    const isLocked = lockedElements.simpleLock.some((lockedPath) =>
+      EP.pathsEqual(lockedPath, selectablePath),
+    )
+    const isFragment = MetadataUtils.isElementPathFragmentFromMetadata(
+      componentMetadata,
+      selectablePath,
+    )
+    const isConditional = MetadataUtils.isElementPathConditionalFromMetadata(
+      componentMetadata,
+      selectablePath,
+    )
+    const isRootPath = EP.isRootElementOfInstance(selectablePath)
+
+    const mustReplaceWithChildren =
+      isLocked ||
+      (!isFeatureEnabled('Fragment support') && isFragment) ||
+      (!isFeatureEnabled('Conditional support') && isConditional)
```

Comment:
I think this should be removed. fragments and conditionals should be selectable on the canvas: they are groups now!

Reply:
ahhhh ok, I missed the `!`. ok now it makes sense!